### DISCUSSION TITLE
UX Issue - Midnight Theme - Line breaks are invisible

### DISCUSSION THREAD

**Discussion by tiddesk** [standardnotes/forum#2881] [General]
**Describe the bug**

**To Reproduce**
1. Activate Midnight Theme
2. Add line break in Alternative Rich text
3. See no line break

**Expected behavior**
The line breaks should be in a color that matches the text or contrasts more sharply with the background

**Desktop (please complete the following information):**
 - OS: Win 11
 - Browser Firefox
 - Version 3.23.218

**Comment by effieeee** [comment]
Can confirm this on dekstop and web 😞
